src/gamecontroller.cs

```diff
@@ -2608,28 +2770,27 @@ interface GCSteeringWheelElement : GCAxisElement {
 		float MaximumDegreesOfRotation { get; }
 	}
 
-	// There are issues with the Generic Types listed here: https://github.com/xamarin/xamarin-macios/issues/15725
-	// [iOS (16,0), Mac (13,0), TV (16,0), MacCatalyst (16,0)]
-	// [BaseType (typeof (NSObject))]
-	// [DisableDefaultCtor]
-	// interface GCPhysicalInputElementCollection<KeyIdentifierType, ElementIdentifierType> // : INSFastEnumeration // # no generator support for FastEnumeration - https://bugzilla.xamarin.com/show_bug.cgi?id=4391
-	// 	where KeyIdentifierType : IGCPhysicalInputElementName /* NSString */ // there's currently not an conversion from GCPhysicalInputElementName, GCButtonElementName, and GCDirectionPadElementName to NSString
-	// 	where ElementIdentifierType : IGCPhysicalInputElement /* id<GCPhysicalInputElement>> */
-	// {
-	// 	[Export ("count")]
-	// 	nuint Count { get; }
-
-	// 	[Export ("elementForAlias:")]
-	// 	[return: NullAllowed]
-	// 	IGCPhysicalInputElement GetElement (string alias);
+	[iOS (16, 0), Mac (13, 0), TV (16, 0), MacCatalyst (16, 0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface GCPhysicalInputElementCollection<KeyIdentifierType, ElementIdentifierType> : INSFastEnumeration // # no generator support for FastEnumeration - https://bugzilla.xamarin.com/show_bug.cgi?id=4391
```

Comment:
Does this comment still stand? Unfortunately bugzilla is nowhere to be found nowadays unless you use https://web.archive.org/web/20170630211645/https://bugzilla.xamarin.com/show_bug.cgi?id=4391 which I find quite surprising!

Reply:
Yes, that comment still stands. I re-filed it here: https://github.com/dotnet/macios/issues/22516